builtin/logical/transit/stepwise_test.go

```diff
@@ -194,6 +194,9 @@ func testAccStepwiseDecrypt(
 		Path:      "decrypt/" + name,
 		Data:      decryptData,
 		Assert: func(resp *api.Secret, err error) error {
+			if err != nil {
```

Comment:
While I agree, this is probably the fix that makes the most sense, this test is now incongruent with all the other tests that we ignore the passed in error into the Assert function. 

Could we instead rename the err variable on line 208 or apply this fix to all the other tests within this file?